config/lib/express.js

```diff
@@ -102,6 +102,12 @@ module.exports.initLocalVariables = function(app) {
     // https://expressjs.com/en/api.html#req.path
     res.locals.canonicalUrl = res.locals.hostPort + req.path;
 
+    // Native mobile app wrapper loads the site with `?app` URL query argument.
+    res.locals.isNativeMobileApp = Object.prototype.hasOwnProperty.call(
```

Comment:
We have lodash `has` here presumably? Would look nicer/neater :)

Reply:
Ahh yes, you're right. I was too sleepy to think. :-)